src/content.ts

```diff
@@ -999,6 +999,16 @@ function getYouTubeVideoIDFromURL(url: string): string | boolean {
             return false;
         }
     }
+
+    interface ytPlayerWindow extends Window {
```

Comment:
It won't get here since searchParams has v, but id.length is not 11

Reply:
moved to top, since it should be the most reliable if it does exist